src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -527,6 +527,11 @@ def _forward(self, model_inputs, return_timestamps=False, generate_kwargs=None):
             generate_kwargs["return_timestamps"] = return_timestamps
             if return_timestamps == "word":
                 generate_kwargs["return_token_timestamps"] = True
+
+                stride = model_inputs.get("stride")
```

Comment:
Only comment is that we end up checking for the stride thrice:
-[ here ](https://github.com/huggingface/transformers/pull/25607/files#diff-ccc8e98fcaa81fdf6317a652438a309bcede0bbe336774288c2fcf91d9f11082R566)

- and [here](https://github.com/huggingface/transformers/pull/25607/files#diff-ccc8e98fcaa81fdf6317a652438a309bcede0bbe336774288c2fcf91d9f11082R571)

so let's just do it once and forall 😉

Reply:
Resolved in https://github.com/huggingface/transformers/pull/25607/commits/0694dbe3e597daf973e3f1545b9fcc2e0da4afdf